src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -71,11 +71,13 @@ import { exec } from "child_process";
 export interface NetEmitterOptions {
     outputFile: string;
     logFile: string;
+    runCodegen: 'true' | 'false';
 }
 
 const defaultOptions = {
     outputFile: "cadl.json",
-    logFile: "log.json"
+    logFile: "log.json",
+    runCodegen: "true"
```

Comment:
Can we find a better name to describe it, 'GenerateSDK'?

Reply:
I personally think that a boolean option with a `true` as default is not very intuitive. Could we flip this switch to something like `skipCodegen` or `skipSDKGeneration`?